assets/app/view/game/map.rb

```diff
@@ -14,6 +15,9 @@ class Map < Snabberb::Component
       needs :selected_route, default: nil, store: true
       needs :selected_company, default: nil, store: true
       needs :opacity, default: nil
+      needs :show_coords, default: Lib::Storage['show_coords'], store: true
```

Comment:
I'm not entirely sure about this.

This will work differently than other player preferences which are stored on the server.

Maybe there should be an option on the profile, and the local option is just a temporary option

Reply:
I thought someone might want to hide the names and leave them hidden, but yeah `localStorage` just applies to the current device.

I feel like it would be fine to have the buttons by the map use (and update) the server player preferences instead of `localStorage`, seems simpler that way.